source/geometries/NextDemoSiPMBoard.h

```diff
@@ -52,6 +54,8 @@ namespace nexus {
     G4bool sipm_verbosity_;
     G4bool visibility_;
     G4bool sipm_visibility_;
+    G4bool sipm_coating_;
+    G4bool sipm_coat_thick_;
```

Comment:
This is not a boolean!

Reply:
Good catch!

Reply:
This is not solved in the last commit

Reply:
Something happend while doing the commit, because they're indeed changed in my local version. I'll do another one and push it. Sorry.